gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -4340,9 +4340,9 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
         stop
       endif
 
-      if (Model%lsm == Model%lsm_noahmp .and. Model%iopt_snf == 4) then
-        if (Model%imp_physics /= Model%imp_physics_gfdl) stop 'iopt_snf == 4 must use GFDL MP'
-      endif
+!      if (Model%lsm == Model%lsm_noahmp .and. Model%iopt_snf == 4) then
```

Comment:
Does this mean noahmp can be used by other MP schemes?

Reply:
According to @panll, yes - Linlin tested NoahMP LSM with Thompson MP for the RRFS applications.